packages/vitest/src/types/config.ts

```diff
@@ -233,14 +233,14 @@ export interface UserConfig extends InlineConfig {
   /**
    * Run tests that cover a list of source files
    */
-  findRelatedTests?: string[] | string
+  relatedSources?: string[] | string
 }
```

Comment:
Should we also call this option `related`? Both watch and run commands have an option with the same name.

Reply:
agreed